crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net.rs

```diff
@@ -736,6 +845,8 @@ impl MetaNet {
                                     rem_url: rem_cli_url,
                                     res: res_store2.clone(),
                                     tun: tuning_params2.clone(),
+                                    metric_msg_out_byte: metric_msg_out_byte2.clone(),
+                                    metric_msg_out_time: metric_msg_out_time2.clone(),
```

Comment:
Is it not possible to create these closer to where they're used? This is a lot of passing

Reply:
I guess I'm a little fuzzy on the intentions of the otel api. If I did that, it'd work fine for InfluxDB since they'd end up being recorded into the same metrics buckets, but maybe another impl would not? Also it'd introduce extra memory overhead if the same metric reporter was having to be instantiated multiple times for every connection.

Reply:
Okay, switched them to static once_cells so we can just use them from anywhere.

Reply:
Works for me, easier to move things around this way and the code can focus on what it's doing until it wants to report something